### PR TITLE
Main implementation - 1st version

### PR DESCRIPTION
**Pull request by JKubicaLedzion**
Main implementation - draft.

### CODE REVIEW

File: Udemy/src/com/udemy/section9/bank/Bank.java [hunk 1]
```diff
@@ -16,50 +16,57 @@ public Bank(String bankName) {
     }
 
     public boolean addBranch (String branchName) {
-        if (findBranch(branchName) != null) {
+        if (findBranch(branchName) == null) {
```
Comment (limak2) [inline Udemy/src/com/udemy/section9/bank/Bank.java]: Please use Objects.isNull

File: Udemy/src/com/udemy/section9/bank/Bank.java [hunk 2]
```diff
@@ -16,50 +16,57 @@ public Bank(String bankName) {
...
             branchList.add(new Branch(branchName));
+            System.out.println("Branch " + branchName + " opened for bank " + bankName);
+            return true;
         }
         return false;
     }
 
     public boolean addCustomer (String branchName, String name, Double initialAmount) {
         Branch branch = findBranch(branchName);
         if (branch != null) {
```
Comment (limak2) [inline Udemy/src/com/udemy/section9/bank/Bank.java]: Please use Objects.nonNull()

File: Udemy/src/com/udemy/section9/bank/Bank.java [hunk 3]
```diff
@@ -16,50 +16,57 @@ public Bank(String bankName) {
...
         }
         return false;
     }
 
-    public void showCustomerList (String branchName){
+    public void showCustomerList (String branchName) {
         Branch branch = findBranch(branchName);
-        if (branch != null){
-            System.out.println("Branch " + branch.getBranchName() + " has customers:");
-            for (int i=0; i <= branch.getCustomerList().size(); i++){
```
Comment (limak2) [inline Udemy/src/com/udemy/section9/bank/Bank.java]: formatting => i = 0

Comment (JKubicaLedzion) [reply Udemy/src/com/udemy/section9/bank/Bank.java]: isNull and nonNUll added

**Pull request merged**